README.md

```diff
@@ -40,15 +47,43 @@ stages:
          group: 0
 ```
 
+When you want to register node in Rancher Manager and create local plan from there:
```

Comment:
I would link this to the elemental-operator readme... something like:
```suggestion
You can create a local plan via a [MachineRegistration CRD](https://github.com/rancher/elemental-operator#machineregistration) , used to register a node in Rancher Manager
```
but it is just a nit, feel free to ignore this!